interactive_ai/workflows/project_ie/job/usecases/data_redaction_usecase.py

```diff
@@ -408,6 +408,22 @@ def remove_lock_in_mongodb_doc(doc: dict) -> dict:
             doc.pop(key, None)
         return doc
 
+    @staticmethod
+    def purge_model_docs_if_necessary(doc: dict) -> dict:
+        purge_info = doc.get("purge_info")
+        if purge_info is not None and purge_info["is_purged"]:
+            return doc
+
+        new_purge_info = {
+            "is_purged": True,
+            "purge_time": now(),
+            "user_id": "export_project_job",
```

Comment:
The field is called `user_uid`:
https://github.com/open-edge-platform/geti/blob/0c5475688cfc59eb5133be12016ff97cdea48501/interactive_ai/libs/iai_core_py/iai_core/repos/mappers/mongodb_mappers/model_mapper.py#L129